Features/Context/PlatformUI.php

```diff
@@ -0,0 +1,327 @@
+
+class PlatformUI extends Context
+{
+    use SubContext\CommonActions;
+
+    /**
+     * PlatformUI relative URL path
+     *
+     * @var string
+     */
+    private $platformUiUri;
+    protected $shouldBeLoggedIn;
+    protected $driver;
+
+    /**
+     * @Given I go to homepage
+     */
+    public function goToPlatformUi()
+    {
+        $this->visit( $this->platformUiUri );
+    }
+
+    /**
+     * @Given I go to PlatformUI app with username :user and password :password
+     */
+    public function goToPlatformUiAndLogIn( $username, $password )
+    {
+        // Given I go to PlatformUI app
+        $this->goToPlatformUi();
+        //wait fos JS
+        $this->waitForJs();
+        // And I fill in "Username" with "admin"
+        $this->fillFieldWithValue( 'Username', $username );
+        //And I fill in "Password" with "publish"
+        $this->fillFieldWithValue( 'Password', $password );
+        //And I click on the "Login" button
+        $this->iClickAtButton( 'Login' );
+        //wait fos JS
+        $this->waitForJs();
+        //Then I should be logged in
+        $this->iShouldBeLoggedIn();
+        //Catches Js errors
+        $this->activateJsErrorHandler();
+    }
+
+    /**
+     * @Given I click (on) the logo
+     */
+    public function iClickLogo()
+    {
+        $this->clickLogo();
+    }
+
+    /**
+     * @Given I click (on) the tab :tab
+     */
+    public function iClickTab( $tab )
+    {
+        $this->clickTab( $tab );
+    }
+
+     /**
+     * @Given I click (on) the menu zone :zone
+     */
+    public function iClickMenuZone( $zone )
+    {
+        $this->clickMenuZone( $zone );
+    }
+
+    /**
+     * @Given I over (on) the menu zone :zone
+     */
+    public function iOverMenuZone( $zone )
+    {
+        $this->overMenuZone( $zone );
+    }
+
+    /**
+     * @Given I click (on) the menu link :link
+     */
+    public function iClickMenuLink( $link )
+    {
+        $this->clickMenuLink( $link );
+    }
+
+    /**
+     * @Given I click (on) the sub-menu option :subMenu
+     */
+    public function iClickSubMenuOption( $subMenu )
+    {
+        $this->clickSubMenuLink( $subMenu );
+    }
+
+     /**
+     * @Given I click (on) the side menu option :sideMenuOption
+     */
+    public function iclickSideMenuOption( $sideMenuOption )
+    {
+        $this->clickSideMenuLink( $sideMenuOption );
+    }
+
+    /**
+     * @Given I click (on) the content type :contentType
+     */
+    public function iclickContentTypeOption( $contentType )
+    {
+        $this->clickContentType( $contentType );
+    }
+
+    /**
+     * @Given I click (on) the content tree with path :path
+     */
+    public function iClickContentTreePath( $path )
+    {
+        $this->clickContentTreeLink( $path );
+    }
+
+    /**
+     * @Given I upload the file :path
+     */
+    public function iUploadTheFile( $path )
+    {
+        $this->attachFilePrepare( $path, '#ez-field--file' );
```

Comment:
this id is weird because it might not be unique across the page, so I guess there's a problem in the generated id in the page which should be something like `#ez-field-42-file` where 42 is the content id.